use-cases/virgo/config.yaml

```diff
@@ -75,31 +75,33 @@ training_pipeline_small:
           validation_proportion: 0.1
           rnd_seed: 42
       - class_path: data.TimeSeriesProcessorSmall          
+        init_args:
+          name: Virgo Small Dataset
       - class_path: trainer.NoiseGeneratorTrainer
         init_args:
-        config:
+          config:
             generator: simple #unet
-            batch_size: ${batch_size}
```

Comment:
shouldn't this be `${batch_size}` rather than 1? As far as I remember, we used 1 only when working on HDF5 data which ca be read in chunks

Reply:
Yes, I think you are correct

Reply:
Very true - I'll change it